site/scripts/library.js

```diff
@@ -404,6 +404,50 @@ function showMangaDetails(manga, categories, source) {
     }
   })();
 
+  ///////////////////
+  const trackingImages = [
+    { src: 'img/trackers/ic_tracker_mal.webp', greyed: true },
+    { src: 'img/trackers/ic_tracker_anilist.webp', greyed: true },
+    { src: 'img/trackers/ic_tracker_kitsu.webp', greyed: true },
+    { src: 'img/trackers/ic_manga_updates.webp', greyed: true },
+    { src: 'img/trackers/ic_tracker_shikimori.webp', greyed: true },
+    { src: 'img/trackers/ic_tracker_bangumi.webp', greyed: true },
+  ];
```

Comment:
Move to `constants.js`, no need to redefine every time

Reply:
I dont mind them being external but may of those icons are [Komga Icon](https://external-content.duckduckgo.com/ip3/komga.org.ico) are circular and I'm not yet convinced on making it a circle. Maybe we can find another source for the icons, but IDK